charts/opensearch/templates/statefulset.yaml

```diff
@@ -354,7 +354,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
-        {{- if (and (has "master" .Values.roles) (not .Values.singleNode)) }}
+        {{- if (and (or (has "master" .Values.roles) (has "cluster_manager" .Values.roles)) (not .Values.singleNode)) }}
```

Comment:
Good to add description in README.md as well for the same. Also echoing on the question by @prudhvigodithi how is it impacting and what are the failures we are seeing. Good to capture it in the github issue

Reply:
I can provide some input on the question posed by @prudhvigodithi . I recently was deploying Opensearch for the first time and was having issues when there was more than one replica. I had been attempting to use all inclusive language, to include specifying a new `roles` key in my `values.yaml` where I replaced the `master` role with `cluster_manger`. I tracked down the issue to these conditional renderings in the helm chart. I solved it by just using the non-inclusive language.